tests/pyincore/analyses/indp/test_indp.py

```diff
@@ -28,7 +28,7 @@
 def run_with_base_class():
     dev_client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
     prod_client = IncoreClient()
-    dev_datasvc = DataService(dev_client)
+    prod_datasvc = DataService(prod_client)
```

Comment:
Just following up here, when you get a chance, can you change this back to using data on dev? I think it's better if tests only use our dev instance, even if these aren't automatically ran.

Reply:
Yes I was planning to do that but I lost track on this. Will do shortly!

Reply:
It took me longer than I think to migrate all the datasets and mappings but they are done~~